pkg/ssh/shell/buildLoadTest.sh

```diff
@@ -2,6 +2,9 @@
 # get most updated load test repo
 echo "getting load test repo ..."
 git -C {{ .LoadTestRepoDir }} pull || git clone {{ .LoadTestRepo }}
+{{if .CheckoutCommit }}
```

Comment:
do we need `.CheckoutCommit`  if we can check `.LoadTestGitCommit` is not empty ?